### PR TITLE
Create Swarmflow class to represent a workflow object

### LINKED ISSUE #3: Create a Swarmflow object
Need to implement a Fireworks Workflow like object with some additional functionality.

- Workflow need to have a workflow id and a firework costs attributes

- Need to modify the functionality of some methods of the Fireworks Workflow class

### PR DESCRIPTION
**Pull request by AyeshW**
- This PR resolves #3 

- Swarmflow class inherits the Fireworks's Workflow class.

- Swarmflow has an ID to uniquely identify the Swarmflow and fireworks' costs attributes.

- It overrides _reassign_ids, to_db_dict, from_Fireworks methods.

Script Path: 
"swarmform/core/swarmwork.py"

### CODE REVIEW

File: swarmform/core/swarmwork.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    def to_db_dict(self):
+        m_dict = super().to_db_dict()
+        m_dict['sf_id'] = self.sf_id
+        return m_dict
+
+    @classmethod
+    def from_Firework(cls, fw, name=None, metadata=None, sf_id=None):
+        """
+        Return Workflow from the given Firework.
```
Comment (KalanaDananjaya) [inline swarmform/core/swarmwork.py]: Have to change to "Return Swarmflow ... "

File: swarmform/core/__init__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# These imports allow a much simpler import of core Fireworks functionality.
+# E.g., you can now do "from fireworks import Firework", instead of from
+# "fireworks.core.firework import Firework".
+from fireworks.core.firework import FiretaskBase, FireTaskBase, Firework, Launch, Workflow, \
+    FWAction, Tracker
+from fireworks.core.fworker import FWorker
+from fireworks.core.launchpad import LaunchPad
+from fireworks.utilities.fw_utilities import explicit_serialize
+from fireworks.user_objects.firetasks.script_task import ScriptTask, PyTask
```
Comment (KalanaDananjaya) [inline swarmform/core/__init__.py]: Are we using these imports? Looks like we are not using many of them

File: swarmform/core/__init__.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# These imports allow a much simpler import of core Fireworks functionality.
```
Comment (KalanaDananjaya) [inline swarmform/core/__init__.py]: Shall we remove these comments?

**Review by KalanaDananjaya — CHANGES_REQUESTED** [review]

File: swarmform/core/swarmwork.py (hunk 1 above)
Comment (AyeshW) [reply swarmform/core/swarmwork.py]: Fixed

File: swarmform/core/__init__.py (hunk 2 above)
Comment (AyeshW) [reply swarmform/core/__init__.py]: Fixed

**Comment by KalanaDananjaya** [comment]
Looks good to me

**Review by rpjayasekara — APPROVED** [review]

**Pull request merged**